cypress/integration/register.spec.js

```diff
@@ -51,12 +51,14 @@ describe('Teste Automatizado', function (){
       .select("14")
 
     cy.get('input[id="firstpassword"]') //senha
-      .type("12345678")
+      .type("Ertar1345%")
    
     cy.get('input[id="secondpassword"]') //confirmar senha
-      .type("12345678")   
+      .type("Ertar1345%")   
 
     cy.get('button[id="submitbtn"]').click()
+
+    cy.url().should('include', '/WebTable.html') //verifica se a nova URL tem /WebTable.html
```

Comment:
aqui, no final do teste ele redireciona para outra página que tem a URL http://demo.automationtesting.in/WebTable.html, então coloquei o assert para verificar se foi para a página certa. No Cypress ele tá dando um 404 nessa URL, eu preciso olhar com calma pra entender. Mas assim o teste já passa!